### PR TITLE
Publish cloud.gov Pages CRM

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:
-
- Add new CRM file for Pages
- Add a news post
- Update FedRAMP tracker with links to both
- Clarify FedRAMP® tracker
-

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/peterb/new-crm)

## Security Considerations
We have previously cleared our CRM as open-source

### CODE REVIEW

File: _docs/overview/fedramp-tracker.md [hunk 1]
```diff
@@ -38,7 +38,16 @@ Here's an example of a control breakdown for a simple moderate-impact system hos
...
+### Control Implementation Summary (CIS) + Customer Responsibility Matrix (CRM):
+
+We publish two CIS/CRM documents, one for the Paas/Platform service and one for the Pages service:
+
+* [cloud.gov PaaS CIS Worksheet]({{ site.baseurl }}/resources/cloud.gov-CIS-Worksheet.xlsx) summarizes each Low and Moderate security control and whether it is handled by cloud.gov (inheritable), a shared responsibility, or a customer responsibility. It includes guidance on which controls a customer on the Platform can fully or partially inherit from cloud.gov.
+  * Last Update: 2022-11-15 - Updated front matter
+* [cloud.gov Pages CIS Worksheet]({{ site.baseurl }}/resources/cloud.gov-CIS-Pages-Worksheet.xlsx) summarizes each Low and Moderate security control and whether it is handled by cloud.gov (inheritable), a shared responsibility, or a customer responsibility. It includes guidance on which controls a customer on Pages can fully or partially inherit from cloud.gov.
```
Comment (drewbo) [inline _docs/overview/fedramp-tracker.md]:
```suggestion
* [cloud.gov Pages CIS Worksheet]({{ site.baseurl }}/resources/cloud.gov-Pages-CIS-Worksheet.xlsx) summarizes each Low and Moderate security control and whether it is handled by cloud.gov (inheritable), a shared responsibility, or a customer responsibility. It includes guidance on which controls a customer on Pages can fully or partially inherit from cloud.gov.
```

File: _posts/2022-11-15-pages-crm-release.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Release of cloud.gov Pages Customer Responsibility Matrix (CRM)
+date: November 15, 2022
+excerpt: As part of the launch of cloud.gov Pages we are 
+  publishing the Pages-specific Customer Responsibility Matrix (CRM)
+---
+
+Our website publishing platform, [cloud.gov
+Pages](https://cloud.gov/pages/), is now fully authorized by
```
Comment (drewbo) [inline _posts/2022-11-15-pages-crm-release.md]:
```suggestion
Pages]({{ site.baseurl }}pages/), is now fully authorized by
```

File: _posts/2022-11-15-pages-crm-release.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+[FedRAMP®](https://fedramp.gov) at the Moderate impact level, and
+is replacing the Federalist offering, formerly at
+https://federalist.18f.gov. As part of that migration, we are
+publishing the **[Control Implementation Summary (CIS) + Customer Responsibility Matrix (CRM) + Control-by-Control Inheritance (.xlsx)]({{ site.baseurl }}resources/cloud.gov-Pages-CIS-Worksheet.xlsx)**.
```
Comment (drewbo) [inline _posts/2022-11-15-pages-crm-release.md]:
```suggestion
publishing the **[Control Implementation Summary (CIS) + Customer Responsibility Matrix (CRM) + Control-by-Control Inheritance (.xlsx)]({{ site.baseurl }}/resources/cloud.gov-Pages-CIS-Worksheet.xlsx)**.
```

File: _posts/2022-11-15-pages-crm-release.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+The CRM is a summary of each Low security control and whether it
+is handled by cloud.gov, a shared responsibility, or a customer
+responsibility. It includes guidance on which controls a customer
+system can fully or partially inherit from cloud.gov Pages.
+
+We will release a CRM for the cloud.gov Pages Moderate impact controls in the near future. Updates to the cloud.gov Pages CRM will
+be reflected on our [FedRAMP Tracker page]({{ site.baseurl }}/overview/fedramp-tracker/)
```
Comment (drewbo) [inline _posts/2022-11-15-pages-crm-release.md]:
```suggestion
be reflected on our [FedRAMP Tracker page]({{ site.baseurl }}/docs/overview/fedramp-tracker/)
```

**Review by drewbo — CHANGES_REQUESTED** [review]
Thanks for releasing @pburkholder! I made a few changes to fix the links. Also this PR includes the commit from #2206 but is fine to merge if that goes in first. Another note, I think slashes in the branch name break the preview urls so once you make these changes, I'll pull locally again to confirm everything looks good

**Comment by pburkholder** [comment]
I updated the PREVIEW link. Ready to :shipit: ?

**Review by apburnes — APPROVED** [review]
💯

**Review by drewbo — APPROVED** [review]

**Pull request merged**